src/components/themed/AddressTile.js

```diff
@@ -53,16 +53,22 @@ const isPaymentProtocolUri = (parsedUri: EdgeParsedUri): boolean => {
 }
 
 class AddressTileComponent extends React.PureComponent<Props, State> {
+  handleAppStateChange: (nextAppState: AppState) => void
+
   constructor(props: Props) {
     super(props)
 
     this.state = {
       clipboard: '',
       loading: false
     }
+
+    this.handleAppStateChange = this.handleAppStateChange.bind(this)
```

Comment:
Try defining this as an arrow function; if you can't get it to work, then go with this.